core/email.py

```diff
@@ -171,6 +222,7 @@ def email_to_admin(
     if not username and not user_email:
         username, user_email = sendto, sendto_email
     elif not user_email:  # Username provided
+        # TODO: Pass only strings, avoid passing 'User' object here.
```

Comment:
Agree! [P.O.L.A.](http://c2.com/cgi/wiki?PrincipleOfLeastAuthority) _(plus some [yelling about the topic](https://www.youtube.com/watch?v=eL5o4PFuxTY))_